src/compas/datastructures/mesh/_mesh.py

```diff
@@ -2633,6 +2633,32 @@ def face_area(self, fkey):
         """
         return area_polygon(self.face_coordinates(fkey))
 
+    def face_circle(self, fkey):
```

Comment:
not sure if it is advisable to add trimesh specific functions to the general mesh implementation. perhaps we could make this the rule: if N-gon if can be added directly, if trimesh/quadmesh specific => add as separate function...